Source/Engines/ModuleEngineConfigs.cs

```diff
@@ -261,13 +319,22 @@ public override void OnStart(StartState state)
             if (HighLogic.LoadedSceneIsEditor)
                 GameEvents.onPartActionUIDismiss.Add(OnPartActionGuiDismiss);
 
+            InitializeB9PSIntegrationIfEnabled();
+
             ConfigSaveLoad();
 
             SetConfiguration();
 
             // Why is this here, if KSP will call this normally?
             part.Modules.GetModule("ModuleEngineIgnitor")?.OnStart(state);
         }
+
+        public override void OnStartFinished(StartState state)
+        {
+            base.OnStartFinished(state);
+
+            HideB9PSVariantSelector();  // Just the one managed by RF, if there is one.
```

Comment:
It's a well-known fact in some circles that DRVeyl hates whitespace.
`PartModule.OnStart` / `PartModule.OnStartFinished` and similar methods are empty, so calling `base.OnStartFinished` from something inheriting directly from PartModule will always be an empty return.

Reply:
> PartModule.OnStart / PartModule.OnStartFinished and similar methods are empty, so calling base.OnStartFinished from something inheriting directly from PartModule will always be an empty return.

Thank you, I was not aware.